template/index.js

```diff
@@ -34,14 +34,12 @@ module.exports = yeoman.generators.NamedBase.extend({
         }
     },
 
-    writing: function files() {
+    files: function files() {
         debug("creating template '%s'", this.name);
         this.fs.copyTpl(
             this.templatePath('template.dust'),
             this.destinationPath(path.join('public', 'templates', this.name + '.dust')),
-            {
-                jsModule: this.options.jsModule
-            }
+            this.config.getAll()
```

Comment:
Just the usual tension between brevity and being explicit; I've always favored explicit more -- "how did this thing end up in this namespace? Because I put it there."